frappe/utils/change_log.py

```diff
@@ -182,32 +185,53 @@ def parse_latest_non_beta_release(response):
 
 	return None
 
-def check_release_on_github(app):
-	# Check if repo remote is on github
-	from subprocess import CalledProcessError
+
+def check_release_on_github(app: str):
+	"""
+	Check the latest release for a given Frappe application hosted on Github.
+
+	Args:
+		app (str): The name of the Frappe application.
+
+	Returns:
+		tuple(Version, str): The semantic version object of the latest release and the
+			organization name, if the application exists, otherwise None.
+	"""
+
+	from giturlparse import parse
+	from giturlparse.parser import ParserError
+
 	try:
-		remote_url = subprocess.check_output("cd ../apps/{} && git ls-remote --get-url".format(app), shell=True).decode()
-	except CalledProcessError:
-		# Passing this since some apps may not have git initializaed in them
-		return None
+		# Check if repo remote is on github
+		remote_url = subprocess.check_output("cd ../apps/{} && git ls-remote --get-url".format(app), shell=True)
+	except subprocess.CalledProcessError:
+		# Passing this since some apps may not have git initialized in them
+		return
 
 	if isinstance(remote_url, bytes):
 		remote_url = remote_url.decode()
 
-	if "github.com" not in remote_url:
-		return None
+	try:
+		parsed_url = parse(remote_url)
+	except ParserError:
+		# Invalid URL
+		return
+
+	# Get latest version from Github
+	if parsed_url.protocol == "http":
```

Comment:
Is there any reason for this? I don't see any reason why `http` URLs should be excluded

Everything else looks good 👍🏼

Reply:
I didn't get it either, but it was already there before and I just included it in the new changes to not break any flows :sweat_smile: